src/_common/theme/theme.ts

```diff
@@ -106,6 +106,11 @@ export class AppTheme extends Vue {
 				${genDarkVar('link', theme.darkHighlight_)}
 				${copyDarkVar('link-hover', 'white')}
 			}
+
+			.scroll-scroller {
```

Comment:
I think instead of making this a global thing of "themes" we should do it just within the scroller component. We shouldn't be setting scroller styling in the theme component. Copy the theme stuff into the scroller component instead.